placeos

```diff
@@ -247,7 +247,19 @@ start_environment() {
     echo "PLACE_EMAIL=${PLACE_EMAIL}" >${EMAIL_ENV}
     # TODO: use init check instead of writing the password.
     echo "PLACE_PASSWORD=${PLACE_PASSWORD}" >>${EMAIL_ENV}
-    
+
+    if [[ $ENABLE_ELASTIC_STACK == "true" ]]; then
```

Comment:
This is missing a cli argument

Reply:
added in https://github.com/place-labs/partner-environment/pull/49/commits/880c69766525a99a37c1acb43a7b9e46efa41cfd